src/tasks/actions.js

```diff
@@ -16,7 +16,8 @@ import {
 } from './action-types';
 
 
-export function createTask(task, cb = (t)=>{}) {
+export function createTask(task, user, cb = (t)=>{}) {
```

Comment:
`createTask` the name doesn't comply with the logic..
this func does add users to the task and adds new task to the list ?

Could be much readable like that:
```
export function createTask(task, user, cb = (t) => {}) {
  task.listeners = addUserToListeners(task, user);

  return (dispatch) =>
    taskList.push(task)
      .then(cb)
      .catch(error => dispatch(createTaskError(error)));
}
```